common/src/main/java/com/microsoft/identity/common/internal/util/PackageUtils.java

```diff
@@ -134,15 +134,14 @@ public static List<X509Certificate> readCertDataForApp(final String packageName,
      * @throws CertificateEncodingException if a certificate was corrupt.
      * @throws ClientException if no valid hash was found in the list.
      */
-    public static final String verifySignatureHash(final @NonNull List<X509Certificate> certs,
+    public static String verifySignatureHash(final @NonNull List<X509Certificate> certs,
                                              final @NonNull Iterator<String> validHashes)
```

Comment:
nit: either rename the method or rename the variable (to make it explicit that we're validating againts sha 512 hashes only.

Reply:
Renamed variable inside method: cddee2f39f960e1a1c9064564b56ab01962a30c7